variantlib/plugins/_subprocess.py

```diff
@@ -8,10 +8,16 @@
 from itertools import groupby
 from typing import TYPE_CHECKING
 
-from _variantlib_protocols import PluginType
-from _variantlib_protocols import VariantFeatureConfigType
-from _variantlib_validators_base import ValidationError
-from _variantlib_validators_base import validate_type
+from _variantlib_protocols import PluginType  # type: ignore[import-not-found]
```

Comment:
Not really a request but I wonder if we could get around this by using regular imports and substituting them while writing.